neon/hardware/compatible-devices/index.md

```diff
@@ -1,11 +1,11 @@
 # Companion Device
 The Companion device is a flagship Android smartphone. It is a regular phone that is not customized or modified in any way. To ensure maximum stability and performance we can only support a small number of carefully selected and tested models. The Neon Companion app is tuned to work with these particular models as we require full control over various low-level functions of the hardware. 
 
-The models we currently support are OnePlus 8, OnePlus 8T, OnePlus 10 Pro, and Motorola Edge 40 Pro.
+The model we currently support are Motorola Edge 40 Pro.
```

Comment:
Typo fix

```suggestion
The model we currently support is Motorola Edge 40 Pro.
```

Reply:
Nonetheless, I would phrase it like we recommend, rather supported, the other ones are also still supported.

Reply:
```suggestion
The currently supported model is the Motorola Edge 40 Pro. Previously supported models are the OnePlus 8, OnePlus 8T, and OnePlus 10 Pro. 
```